pyproject.toml

```
@@ -86,6 +86,7 @@ kubernetes_bearer_token = "awx_plugins.credentials.plugins:kubernetes_bearer_tok
 registry = "awx_plugins.credentials.plugins:registry"
 galaxy_api_token = "awx_plugins.credentials.plugins:galaxy_api_token"
 net = "awx_plugins.credentials.plugins:net"
+hcp_terraform = "awx_plugins.credentials.plugins:hcp_terraform"
```

I think the exposed name should avoid abbreviations. It's often difficult to track what they mean, especially for people outside the context.
How about
```suggestion
hashicorp-terraform = "awx_plugins.credentials.plugins:hcp_terraform"
```
for example?

---

Ah, never mind. I google it and this seems to be a part of the official name. Marking the thread as resolved.